### ISSUE TITLE
Compatibility with React Native 0.64

### ISSUE THREAD

**Issue by ianebstein** [opentok/opentok-react-native#477]
# Bug Report

**Current behavior**
Initializing an OpenTok session results in a crash.

**Steps to reproduce**
1. Update your app to React Native 0.64
2. Initialize instance of OpenTok

**What is the current *bug* behavior?**
Full OT.js crash

**What is the expected *correct* behavior?**
Initialization as usual

**Relevant logs and/or screenshots**

```
TypeError: nativeEvents.listeners is not a function
```
![image](https://user-images.githubusercontent.com/80416631/111241323-452b6e80-85d3-11eb-934d-c48748c537ee.png)

## Explanation
The `NativeEventEmitter` export has changed per https://github.com/facebook/react-native/commit/b11d6ecbb8bb2f0d6f423be6775e587f4e9b1c4d

**TLDR**
```diff
-listeners(eventType: string): [EmitterSubscription] { ... }
+listenerCount(eventType: string): number { ... }
```

With this method no longer available, OT.js's `setNativeEvents` function will have to be updated.

When patching this locally I can get past the crash and everything appears to be running as per usual, but I'm definitely worried about what's happening behind the scenes here.

<details>
  <summary>OT.js Patch</summary>

```diff
const setNativeEvents = (events) => {
  const eventNames = Object.keys(events);
  OT.setNativeEvents(eventNames);
  each(events, (eventHandler, eventType) => {
-   const allEvents = nativeEvents.listeners()
+	const allEvents = []
    if (!allEvents.includes(eventType)) {
      nativeEvents.addListener(eventType, eventHandler);
    }
  });
};
```

</details>

**Comment by flogy** [comment]
Receiving the exact same issue. Thanks for the patch!

**Comment by birdwell** [comment]
Any update on when this will be out?

**Comment by mbouxin** [comment]
A patch actually matching what was node before would be 

```
const setNativeEvents = (events) => {
  const eventNames = Object.keys(events);
  OT.setNativeEvents(eventNames);
  each(events, (eventHandler, eventType) => {
    if (nativeEvents.listenerCount(eventType) <= 0) {
      nativeEvents.addListener(eventType, eventHandler);
    }
  });
};

```

**Comment by abdulajet** [comment]
Thanks for the contribution, solved this issue in 0.17.0
